src/editors/textEditor.js

```diff
@@ -286,7 +286,7 @@ class TextEditor extends BaseEditor {
     // TD is outside of the viewport.
     if (!this.TD) {
       if (!force) {
-        this.close();
```

Comment:
I guess these issues are related: #3557, #3656 and definitely to do on a separated branch.